### DISCUSSION TITLE
Carousel Component

### DISCUSSION THREAD

**Discussion by cyonder** [tailwindlabs/headlessui#638] [Ideas]
It would have been very nice to have a carousel component with indicators and arrows to navigate. Additionally, being able to configure if arrows should appear on the slide or below the slide next to indicators, would be nice too.

**Comment by KApilBD** [comment]
Yes, It would be interesting to see some solutions for Carousel/Slider from headlessui. In addition to the above comment, it would be nice if arrows could be configurable as well, I mean you can place the arrows as you please and be allowed to give custom styles.
